chroma/api/local.py

```diff
@@ -79,8 +79,11 @@ def raw_sql(self, raw_sql):
         return self._db.raw_sql(raw_sql)
 
     def create_index(self, model_space=None, dataset_name=None):
+
+        model_space = model_space or self.get_model_space()
```

Comment:
This is a snuck-in clean up change to match the other handlers way of getting the model space.